src/Tracker/__mocks__/safeWoopra.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable @typescript-eslint/no-empty-function, @typescript-eslint/explicit-module-boundary-types */
+// @TODO: Put proper implementation to test Tracker
```

Comment:
Is there a ticket in the backlog to track this piece of work to do?

Reply:
Nope, with the most basic mock, everything just is a dummy function. I put a `@TODO` here so if anyone wants to test Woopra further, the mocks should go here.

Reply:
Ah, ok. Understood.